test/winter_onboarding_2021/fleet_management/handler/cab_test.clj

```diff
@@ -0,0 +1,10 @@
+(ns winter-onboarding-2021.fleet-management.handler.cab-test
+  (:require [clojure.test :refer [deftest is testing use-fixtures]]
+            [winter-onboarding-2021.fleet-management-service.handlers.cab :as handler]
+            [winter-onboarding-2021.fleet-management.fixtures :as fixtures]))
+
+;; (def json)
+
+;; (deftest list-cabs-handler
+;;   (testing "Handler works properly"
+;;     is (= (handler/get-cabs) json)))
```

Comment:
We could remove this commented out part if it's not getting used.
But if we plan to use it then `Handler works properly` is not very indicative of what it will test. 

Something like `Should return first 10 cabs in an HTML table` would be better, I think.